public/install.php

```diff
@@ -66,6 +66,14 @@
 			<?php endif ?>
 		</tr>
 		<tr>
+			<th>Composer Installed</th>
+			<?php if (is_dir($vendor_path) AND is_file($vendor_path.'autoload.php')): ?>
+				<td class="pass"><?php echo $vendor_path ?></td>
```

Comment:
@gnovaro Kohana 3.4 doesn't support any PHP version less than 5.4 so it's perfectly safe to use and should be favoured over the explicit `echo`.